### ISSUE TITLE
Form a DAO

### ISSUE THREAD

**Issue by ligi** [tincubeth/coordination#1]
create multisig with all members of this org and link to tincubeth.eth that was [registered here](
https://etherscan.io/tx/0x2afcf94eb1a4173b5bb04bce5b5ef88f4c858c25b5edb6da657cc2ed8d24600a)

**Comment by ligi** [comment]
now even more relevant as we won some DAI at flyingcircuit and as a group unanimously decided on forming a DAO to manage these funds and pay for things like:
 * printing stickers/silk screens/.. with our awesome logo
 * fees (e.g. ENS or domain fees)
 * stakes (e.g. to run our own TinCubETH nodes)
 * bounties to improve on TinCubETH
 * ...

The DAO would consist of the four founding team members: @ligi @olios @ltfschoen @eshavkun 
and to get to 5 (5 is a good number for an organisation for a lot of reasons) we would also like to invite the main organizer of the hackathon
My suggestion would be to keep the overhead small and do most of the DAO coordination in this repo and just manage the funds with a 3/5 multisig (hints on which one much welcome - I just know it should not be the parity one and would tend to the gnossis one)
Later on we might move to a real DAO platform like [Colony](https://colony.io).

**Comment by ltfschoen** [comment]
As discussed in brief meeting yesterday with @ligi @eshavkun (cc @olios), we'll try using the Gnosis Multisig and Colony for the DAO, initially using a small amount of funds. Some links below

* https://wallet.gnosis.pm
* https://github.com/gnosis/MultiSigWallet/tree/master/dapp
* https://github.com/gnosis/MultiSigWallet
* https://colony.io/

**Comment by ligi** [comment]
Yes - unfortunately @olios  was in another city - we where @eshavkun @ltfschoen and @ligi
So the idea is to test the multisig first with a small amount. If everything works fine we move all funds there. Then we move a small amount to a colonly instance - experiment for a long time - and when we see it working nice move the whole multisig there.

**Comment by ltfschoen** [comment]
@ligi have you already requested early access to Colony for Tincubeth by filling out this form https://share.hsforms.com/1jjT2B9U1QuuLNDj7azGrSQ2vvap?

**Comment by ltfschoen** [comment]
i've only had time to do some research about Colony so far, which i've summarised here in this gist https://gist.github.com/ltfschoen/5cb83e67a3bdf7212710c8a695fef547

**Comment by ligi** [comment]
@ltfschoen thanks - very helpful. Yes I requested early access but got this:

> Hi ,   I just saw that you requested early access to Colony and wanted to  introduce myself as your main point of contact. When you are ready,  let's find some time to chat: clny.io/auryn.   I am really interested to learn more about none and how you plan to use Colony, perhaps you could give me a quick overview?   In the meantime, here are some resources that might be useful:  The Colony blog - it has lots of awesome articles that dive into Colony at every level Our help center - with answers to our most common questions Our developer portal - in case you feel like diving into the code  Look forward to hearing from you.         Auryn Macmillan Business Development & Success Management      twitter: telegram: github: @auryn_macmillan @auryn_macmillan @auryn-macmillan colony.io \| docs.colony.io |   | Auryn Macmillan Business Development & Success Management |   | twitter: telegram: github: | @auryn_macmillan @auryn_macmillan @auryn-macmillan | colony.io \| docs.colony.io

**Comment by auryn-macmillan** [comment]
Hi everyone,
Auryn from Colony here.
Let me know if you have any questions about Colony, happy to help.

**Comment by ligi** [comment]
@auryn-macmillan great! The most important question is: what do we need to do to be able to try it out (get into the Early Access program).

**Comment by auryn-macmillan** [comment]
@ligi let's set up a time to chat more about it, I would love to hear more about your use-case and help you get your colony deployed.

You can pick a time that suits you at clny.io/auryn.

Cheers!

**Comment by ligi** [comment]
@auryn-macmillan no thanks - I do not want to chat. If this is needed to use colony - I will not use colony.

**Comment by auryn-macmillan** [comment]
It is not necessarily needed. We are just wanting to set up tight feedback loops with the first round of projects that launch colonies, so that we can immediately start to iterate on any issues they run into and the features that they find most useful.

**Comment by ligi** [comment]
why not just a repo (private if need be) where we can open issues? Really think private chats are a waste of time for these cases. Much overhead as information-flow is limited.

**Comment by auryn-macmillan** [comment]
> Much overhead as information-flow is limited

Actually, from our point of view, we get a lot more information from being able to observe people during the whole setup process.
And we find that information flows much more freely in a call than in an asynchronous communication channel.

> why not just a repo (private if need be) where we can open issues?

We already have a [feedback portal](https://clny.io/feedback) where you can post feedback and bugs once you are set up. We just want to be there for the setup process for right now so that we can learn as much as possible about the UX from real world interactions.

**Comment by auryn-macmillan** [comment]
Either way, you can create and use a colony without our consent. We can't (and don't want to be able to) stop people from using colony.
As I mentioned earlier, we are just trying to build feedback loops with our early users.

**Comment by ligi** [comment]
I try to stay in asynchronous communication channels as much as possible. I think synchronous communication channels come with too much overhead. Only use them when not otherwise possible. I actually just need a link to a getting started guide. And if there is non public code needed (as it usually is in early access programs) - I will need access to those.
